### ISSUE TITLE
How to install Git

### ISSUE THREAD

**Issue by CJ009** [talkpython/python-for-absolute-beginners-course#3]
Hello,

It might be a strange question but ive had a hard time installing git in windows (Yes, im stuck in the set up part of lecture). I actually dont undestand whats all the stuff its asking me. Do i just leave everything as is and install it? (Next, Next, Next)
What components should i install? 
PyCharm is the text editor, but i tried to add it in other and the Test Custom editor fails.
Then there is the PATH Selection, the HTTPS, The line ending conversions, etc.

In short, I need a bit of guidence.

In PYCharm I installed the paths too, I hope that was right.

Thanks in advance for the help!

**Comment by mikeckennedy** [comment]
Hi @CJ009 Seems like you got this fixed. You don't ever really need git for this course. You can always just "download as zip" per the big green button here:

<img width="404" alt="Screen Shot 2020-07-17 at 8 37 18 PM" src="https://user-images.githubusercontent.com/2035561/87843960-67ff5a80-c86d-11ea-9233-e5e0c38de8fe.png">

The advantage of cloning via git is you can pull updates if we make them. But that's rare in this case, but common in real development.

**Comment by CJ009** [comment]
Yes I made it work! But I used Sublime text as my editor and I learn how to configure sublime text to read python so win win!
I appreciate the response though professor. Thank you!
